### PR TITLE
Fix requests

### PR DESCRIPTION
**Pull request by coditva**
- These changes will get the api working with the code. The problem was with the ```get``` and ```post``` requests. The data has to be sent asynchronously.
- Also, you need to change ```get``` and ```post``` requests everywhere like it is done in the Users component. I could not figure out the ```fake_api``` shit.

### CODE REVIEW

File: src/Components/Main/App/Content/Users/Users.js [hunk 1]
```diff
@@ -17,9 +17,9 @@ export default class Users extends React.Component {
 	}
 
 	getAllUserData() {
-		let response = this.props.getRequest("http://localhost:2000/users/");
-		console.log(response);
-		return response;
+		this.props.getRequest("http://localhost:2000/users/", (res) => {
```
Comment (coditva) [inline src/Components/Main/App/Content/Users/Users.js:20]: The get and post methods should be used like this. **Everywhere**.

Comment (a-y-u-s-h) [reply src/Components/Main/App/Content/Users/Users.js:20]: It's already correct. I've defined a function which takes URL, sends token along with get request and then returns response asynchronously, and it's named "getRequest". Callback will change depending on what component is firing it, so better just return response and deal with it later.
`getRequest` in some other file. Did it to reduce code duplication across files. 

Looks like this : 

```
getRequest(url) {
		var token = localStorage["aiims-login-token"]
			.replace(/^"/, "")
			.replace(/"$/, "");
		$.ajax({
			url: url,
			type: "GET",
			datatype: "application/json",
			data: { token: token }
		})
			.done(res => {
				return res;
			})
			.fail(err => {
				return err;
			});
	}
```

Comment (a-y-u-s-h) [reply src/Components/Main/App/Content/Users/Users.js:20]: Just need to change it there and send token with url. Query string or something?

Comment (coditva) [reply src/Components/Main/App/Content/Users/Users.js:20]: i modified this very function to include a callback. The callback works. your method, doesn't. sorry. :P

**Pull request merged**